### PR TITLE
[READY] Adds back enclave powerarmor to the T5 armor pool also removes ring.

### PR DESCRIPTION
**Pull request by ErwinSnowden**
The LARP is back, hopefully you get good RNG.
## Description
I removed the one ring from the possibility of getting spawned and added back Enclave PA. This WAS a thing back on BD so I don't know if later on after I left they removed it or what, but it seems redundant since the only place T5 is found is in the enclave bunker.

## Motivation and Context
I don't want a bus item, nor a LOTR reference appearing on normal rounds.
I want to LARP as enclave, I'm sure others do as well. The robust will be able to nab these I suppose?

## How Has This Been Tested?
Should be fine, I made sure to leave no stone unturned when tweaking shit unlike my last PR.

## Screenshots (if appropriate):
N/A even if I did it would just be the enclave bunker with some enclave powerarmor.

## Changelog (necessary)
:cl:
del: Removed the One Ring from rotation/normal gameplay.
add: Enclave Power Armor can once again be found in the Enclave bunker. Not sure why DR removed it.
/:cl:

### CODE REVIEW

File: code/game/objects/effects/spawners/f13lootdrop.dm [hunk 1]
```diff
@@ -345,9 +345,18 @@
...
 				)
+	var/loot3 = list(
+				/obj/item/clothing/suit/armor/f13/power_armor/advanced,
+				/obj/item/clothing/head/helmet/power_armor/advanced
+				)
+	var/loot4 = list(
+				/obj/item/clothing/suit/armor/f13/power_armor/advanced/mk2,
+				/obj/item/clothing/head/helmet/power_armor/advanced/mk2
+				)
+// ^^^^ this could be replaced with tesla if it proves to be troublesome
```
Comment (Afya212) [inline code/game/objects/effects/spawners/f13lootdrop.dm:356]:
```suggestion
	var/loot4 = list(
				/obj/item/clothing/suit/armor/f13/power_armor/t60,
				/obj/item/clothing/head/helmet/power_armor/t60
				)
```
Think having t60 ere in this list would make a better alternative.

**Comment by ErwinSnowden** [comment]
T-60 power-armor only saw large use in the East Coast, it makes sense because it was distributed on the final days of the great war and could only go to more militarized, economically strong areas. (That was the in-lore explanation for the retcon of never seeing them before.)

Fallout 4 is a shit game too so I'd rather not have T-60's appearing and fucking shit up nor do I feel comfortable with assaultrons being in the game since they appear to be additions of DR. If they're not, please correct me.

**Comment by Afya212** [comment]
Ah fair, not familiar much with Fallout 4 because it was trash, but I'm not merging in advance mk2 armor which makes ya practically invulnerable, it's basically the same as having the ring in there.

**Comment by ErwinSnowden** [comment]
It's treason then...

Also hardly invulnerable, just use energy weapons. If you really want I can either make it tesla or adjust the spawning rates so a majority of the time it's just MK I.

**Pull request merged**